File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseCapabilities.java
package app.documents.core.network.manager.models.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import app.editors.manager.mvp.models.base.Base;
import app.editors.manager.mvp.models.base.Capabilities;

public class ResponseCapabilities extends Base {

    @SerializedName(KEY_RESPONSE)
    @Expose
    private Capabilities response;

    public Capabilities getResponse() {
        return response;
    }

    public void setResponse(Capabilities response) {
        this.response = response;
    }

}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseCapabilities.kt
package app.documents.core.network.manager.models.response

import app.documents.core.network.common.models.BaseResponse
import app.documents.core.network.login.models.Capabilities
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ResponseCapabilities(
    @SerializedName(KEY_RESPONSE)
    @Expose
    var response: Capabilities = Capabilities()
) : BaseResponse()